docs/content/Examples-Tutorials-Recipes/Recipes/Access-control/controlling-access-to-cubes-and-views.mdx

```diff
@@ -30,35 +30,63 @@ module.exports = {
 
 ## Data modeling
 
+<CodeTabs>
+
+```yaml
+# Orders.yml
+cubes:
+  - name: orders
+    sql_table: orders
+    public: false
+    # ...
+
+# Users.yml
+cubes:
+  - name: users
+    sql_table: users
+    public: false
+    # ...
+
+# TotalRevenuePerCustomer.yml
+views:
+  - name: total_revenue_per_customer
+    public: COMPILE_CONTEXT.securityContext.isFinance
```

Comment:
```suggestion
    public: "{COMPILE_CONTEXT.securityContext.is_finance}"
```
@keydunov We did agree that this is the preferred way, with quotes and braces, didn't we?

Reply:
We have to keep it without `{}` for now, with Jinja arrival it will be done through the Jinja syntax.